src/Microsoft.ML.Data/Evaluators/Metrics/MultiClassClassifierMetrics.cs

```diff
@@ -17,7 +17,7 @@ public sealed class MultiClassClassifierMetrics
         /// The micro-average metric weighs each class according to the number of instances that belong
         /// to it in the dataset.
         /// </remarks>
-        public double AccuracyMicro { get; }
+        public double MicroAccuracy { get; }
```

Comment:
>MicroAccuracy [](start = 22, length = 13)

Thank you for changing these names. This is way better.

But now you have to reorganize the file to keep it alphabetical :)